bundle-workflow/src/test_workflow/test_result.py

```diff
@@ -0,0 +1,19 @@
+import logging
+
+
+class TestResult:
```

Comment:
This is actually multiple results, so the way this is written mixes responsibilities.  This is what I would do:

- Each test execution returns a `TestResult` that has fields such as `status`, `component` or `config`.
- Each `str(TestResult)` returns that `PASS/FAIL:` string.
- Each TestResult has a `.log` method that chooses `logger.info` vs. `logger.error` to log its `str`.  
- A collection of test results is a `TestResults`, that has a `failed` and/or a `succeeded` property that iterates through its test results and can tell you whether it's pass/fail, returns a Boolean, if that's needed.

Reply:
Thanks @dblock for all the help here. Changed as requested.